xline/Cargo.toml

```diff
@@ -3,14 +3,20 @@ name = "xline"
 version = "0.1.0"
 edition = "2021"
 authors = ["DatenLord <dev@datenlord.io>"]
+description = "Distributed KV storage for metadata"
 repository = "https://github.com/datenlord/Xline"
 readme = "README.md"
-license = "MIT"
+license = "Apache-2.0"
```

Comment:
Why do we need to chagne the license?

Reply:
CNCF recommends Apache 2.0 https://www.cncf.io/blog/2017/02/01/cncf-recommends-aslv2/.